test/integration_test/migration_test.go

```diff
@@ -268,16 +268,37 @@ func extractPortworxDaemonSetFromObjects(objects []runtime.Object) (*appsv1.Daem
 	return pxDaemonSet, remainingObjects
 }
 
-func updateImagesWithKubernetesVersion(objects []runtime.Object) error {
+func updateComponentDeploymentImages(objects []runtime.Object) error {
 	k8sVersion, err := k8sutil.GetVersion()
 	if err != nil {
 		return err
 	}
 	for _, obj := range objects {
-		if dep, ok := obj.(*appsv1.Deployment); ok &&
-			(dep.Name == "stork-scheduler" || dep.Name == "portworx-pvc-controller") {
-			image := strings.Split(dep.Spec.Template.Spec.Containers[0].Image, ":")[0]
-			dep.Spec.Template.Spec.Containers[0].Image = image + ":v" + k8sVersion.String()
+		if dep, ok := obj.(*appsv1.Deployment); ok {
```

Comment:
This is good. We also need to handle images for - 
- ccm-service (px daemonset)
- csi-node-driver-registrar (px daemonset)
- alertmanager (alert manager object)

Reply:
right now we don't have validations for those images, maybe we could add them when we have the validation?